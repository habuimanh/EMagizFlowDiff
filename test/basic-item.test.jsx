const React = require("react")
const basic_items = require("../dist/src/components/views/basic-items")
const renderer = require("react-test-renderer")
// import { ElementContainer } from "../src/components/views/basic-items"

it("correct single item ", () => {
    const singleItem = React.createElement("basic_items.SingleItem", { name: "name", value: 20 })
    expect(singleItem).toMatchSnapshot();
})

it("test single item ", () => {
    const singleItem2 = renderer.create(React.createElement("basic_items.SingleItem", { name: "age", value: 21 }))
    expect(singleItem2).toMatchSnapshot()
})
