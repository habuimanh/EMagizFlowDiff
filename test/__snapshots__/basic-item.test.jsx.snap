// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correct single item  1`] = `
<basic_items.SingleItem
  name="name"
  value={20}
/>
`;

exports[`test single item  1`] = `
<basic_items.SingleItem
  name="age"
  value={21}
/>
`;
